### PR TITLE
INTYGFV-16460: Added aggregation for send

### PR DESCRIPTION
**Pull request by mhornfeldt**

### CODE REVIEW

**Review by ChrilleSigerhed — APPROVED** [review]
Nice and clean, well done!

File: web/src/main/java/se/inera/intyg/webcert/web/csintegration/certificate/SendCertificateFromCertificateService.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        }
+
+        monitoringLogService.logIntygSent(
+            certificate.getMetadata().getId(),
+            certificate.getMetadata().getType(),
+            certificate.getMetadata().getRecipient().getId()
+        );
+        pdlLogService.logSent(certificate);
+
+        return certificate.getMetadata().getId();
```
Comment (mww-sopra-steria) [inline web/src/main/java/se/inera/intyg/webcert/web/csintegration/certificate/SendCertificateFromCertificateService.java:65]: The Webcert-implementation returns OK if successfully sent and returns FAILED is not. Double check the implementation by comparing to the Webcert-facade implementation.

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
See the mapping of Previous and Signed List. When Sent, it should set the status to "SENT".

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
Nice! Works well not in Webcert, but there is another tweak that is necessary.

In CertificateListItemConverterImpl on row 80-82 there needs to be special handling for converting ListIntygEntry to correct status based on the ListType.CERTIFICATES. It should consider the status (matches "SENT") instead of being hardcoded to false (was not introduced in this pr).

**Review by mww-sopra-steria — APPROVED** [review]
Nice! Approved!

**Pull request merged**